test/TestProjectWithSDKRef/TestProjectWithSDKRef.csproj

```diff
@@ -1,5 +1,15 @@
 <Project Sdk="MSBuild.Sdk.SqlProj/#{NBGV_NuGetPackageVersion}#">
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
+        <SqlServerVersion>Sql110</SqlServerVersion>
```

Comment:
That's interesting. Was this change necessary to make it work? I think there's a default somewhere.

Reply:
It's one of the changes I made when I was trying to get the composite project to build. I know I've had issues with that in the past -- build issues when the projects aren't on the same version -- but I'm not sure whether this was actually part of what fixed the problem. I'll experiment with removing it again and see if anything breaks.

Reply:
Looks like it wasn't necessary, I've backed it out.